src/sentry/stacktraces.py

```diff
@@ -0,0 +1,201 @@
+from __future__ import absolute_import
+
+import logging
+
+from sentry.models import Project
+from sentry.utils import metrics
+from sentry.utils.safe import safe_execute
+from collections import namedtuple
+
+
+logger = logging.getLogger(__name__)
+
+
+StacktraceInfo = namedtuple('StacktraceInfo', [
+    'stacktrace', 'container', 'platforms'])
+
+
+class StacktraceProcessor(object):
+
+    def __init__(self, data, stacktrace_infos, project=None):
+        self.data = data
+        self.stacktrace_infos = stacktrace_infos
+        if project is None:
+            project = Project.objects.get_from_cache(id=data['project'])
+        self.project = project
+
+    def close(self):
+        pass
+
+    def preprocess_related_data(self):
+        return False
+
+    def get_effective_platform(self, frame):
+        return frame.get('platform') or self.data['platform']
+
+    def process_frame(self, frame):
+        pass
+
+
+def find_stacktraces_in_data(data):
+    """Finds all stracktraces in a given data blob and returns it
+    together with some meta information.
+    """
+    rv = []
+
+    def _report_stack(stacktrace, container):
+        platforms = set()
+        for frame in stacktrace.get('frames') or ():
```

Comment:
I assume this is so we can interleave native iOS frames w/ JavaScript frames for React Native?

Does this mean there's a new `platform` attr for frames that didn't exist previously?